documentation/source/class.rst

```diff
@@ -366,7 +375,6 @@ This intentionally fails:
 
 It produces the following warning message:
 
-.. warning:: doxygenclass: Cannot find class “made_up_class” in doxygen xml
+.. warning::
+   doxygenclass: Cannot find class “made_up_class” in doxygen xml
```

Comment:
Is there a particular reason for moving these lines into the warning block rather than being on the same line as the `warning`? Does it impact the rendering or semantics at all?

Reply:
It was more of a convention/syntax highlighting fix.

In general, all body of text supplied to directive should go on a new indented line. If the directive takes arguments then that should go on the same line as the directive (for required args) or in the directive's options. RST comments should follow the same convention
```rst
..
    comment goes here (not inline with ``..``)
```

Lastly, rst highlighters have problems with text that was started on the same line as the directive then continued on the next line using the same indent
```rst
.. warning:: This is not a required arg. It is the body which
             is continued with an awkward indent.
```